framework-plugins/lisk-framework-chain-connector-plugin/src/db.ts

```diff
@@ -11,3 +11,41 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+
+import * as createDebug from 'debug';
+import { codec, db as liskDB } from 'lisk-sdk';
+import * as os from 'os';
+import { join } from 'path';
+import { ensureDir } from 'fs-extra';
+import { DB_KEY_CHAIN_CONNECTOR_INFO } from './constants';
+import { chainConnectorInfoSchema } from './schemas';
+import { ChainConnectorInfo } from './types';
+
+const debug = createDebug('plugin:forger:db');
+
+const { Database } = liskDB;
+type KVStore = liskDB.Database;
+
+export const getDBInstance = async (
+	dataPath: string,
+	dbName = 'lisk-framework-chain-connector-plugin.db',
+): Promise<KVStore> => {
+	const dirPath = join(dataPath.replace('~', os.homedir()), 'plugins/data', dbName);
+	await ensureDir(dirPath);
+
+	return new Database(dirPath);
+};
+
+export const getChainConnectorInfo = async (db: KVStore): Promise<ChainConnectorInfo> => {
```

Comment:
I missed to mention to implement `setChainConnectorInfo(ChainConnectorInfo)`. I have updated the issue

Reply:
The set function always needs a key value pair as input
```
Implement setChainConnectorInfo(ChainConnectorInfo)
```
But the updated issue only has a single input parameter.